lib/google/cloud/gemserver/cli/server.rb

```diff
@@ -80,6 +82,10 @@ def deploy
           # Updates the gemserver on a Google Cloud Platform project by
           # redeploying it.
           def update
+            was_deployed = user_input "Has the gemserver been deployed" \
```

Comment:
Actually, I think there is - check for the existence of the config file on Cloud Storage which gets uploaded after a deploy.